base/steer/FairRootManager.h

```diff
@@ -183,7 +183,8 @@ class FairRootManager : public TObject
     /** Replace the time based branch name list*/
     void SetTimeBasedBranchNameList(TList* list);
 
-    void FillEventHeader(FairEventHeader* feh)
+    /** \deprecated Deprecated in v19, will be removed in v20. */
```

Comment:
Isn't it deprecated already in 18.8?

Reply:
Thanks for noticing!

* Added PR #1280 to fix this in 18.8.
* Updated #1275